src/main/java/org/sopt/watchapedia/domain/production/service/ProductionService.java

```diff
@@ -0,0 +1,29 @@
+package org.sopt.watchapedia.domain.production.service;
+
+import lombok.RequiredArgsConstructor;
+import org.sopt.watchapedia.domain.production.dto.response.ProductionDetailResponse;
+import org.sopt.watchapedia.domain.production.dto.response.ProductionGetResponse;
+import org.sopt.watchapedia.domain.production.repository.ProductionRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+@Service
+public class ProductionService {
+    public final ProductionRepository productionRepository;
+
+
+    public List<ProductionGetResponse> getProductions() {
+        return productionRepository.findAll().stream()
+                .map(ProductionGetResponse::of)
+                .toList();
+    }
+
+public ProductionDetailResponse getProductionDetail(final Long productionId) {
+    return productionRepository.findById(productionId)
+            .map(ProductionDetailResponse::of)
+            .orElseThrow();
```

Comment:
예외 상황 시 던져질 예외 클래스 생성이 필요해 보입니다.
```suggestion
            .orElseThrow(() -> new BusinessException(ErrorStatus.ENTITY_NOT_FOUND);
```

Reply:
굿굿입니당